packages/babel-parser/src/types.js

```diff
@@ -565,6 +565,32 @@ export type SequenceExpression = NodeBase & {
   expressions: $ReadOnlyArray<Expression>,
 };
 
+// Pipelines
+
+export type PipelineBody = NodeBase & {
+  type: "PipelineBody",
+};
+
+export type PipelineBareFunctionBody = PipelineBody & {
```

Comment:
I think that in every other place these node types don’t contain "Body"?

Reply:
Also, they can’t extend PipelineBody because the "type" properties are not compatible